scripts/flash/finalize-market.js

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env node
+
+"use strict";
+
+var chalk = require("chalk");
+var getTime = require("./get-timestamp");
+
+function finalizeMarketInternal(augur, marketID, auth, callback) {
+  var marketPayload = { tx: { to: marketID } };
+  augur.api.Market.getFeeWindow(marketPayload, function (err, feeWindowAddress) {
+    if (err) { console.error(err); callback(err); }
```

Comment:
Split into multiple lines.

Reply:
did split into multiple lines for clarity.